bahmni-docker/docker-compose.yml

```diff
@@ -142,11 +137,14 @@ services:
       - openmrsdb
 
   openmrsdb:
-    image: mysql:5.6
+    image: ${OPENMRS_DB_IMAGE_NAME:?}
```

Comment:
I dont mind having an env variable for holding the OPENMRS_DB_IMAGE_NAME but we can push for mysql naked container as default with OPENMRS_DB_CREATE_TABLES set to true. Then in our docs we can mention that if anyone like demo data to have a demo db then they should use the `bahmni/openmrs-db:demo-latest` image refered by `OPENMRS_DB_IMAGE_NAME` instead of naked mysql... This would align to our K8s

Reply:
Hi Nouman, I remember the same discussion around fresh start vs demo data loaded happened in the past as well. It is agreed that since docker-compose is used for local development or for people to explore Bahmni , it is better to start with demo data loaded. This would allow them to use Bahmni out of the box, whereas with fresh start it would need some configurations (Locations, visit types etc) to be done before using it. 

Ref: https://github.com/Bahmni/bahmni-decision-records/blob/main/bahmni-docker/001-use-demo-database-as-default-config.md